tests/test.py

```diff
@@ -1,17 +1,20 @@
-import os
 import requests
 import time
 import pytest
 
-DRS_FILER_URL = os.getenv('DRS_FILER_URL')
+# DRS_FILER_URL = os.getenv('DRS_FILER_URL')

```

Comment:
Please do not comment code - just remove it. If we want to go back, we can make use of version control.